tests/cases/fourslash/codeFixPromiseToAsyncResRef.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path='fourslash.ts' />
+
+////function f():Promise<any> {
+////    return fetch('http://yahoo.com').then(res);
+////}
+////
+////function res(result){
+////    return result.ok;    
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "Convert to use async and await",
+    code: 95055,
+    reportsUnnecessary: true,
+}]);
+
+verify.codeFix({
+    description: "Remove unreachable code",
+    index: 0,
+    newFileContent:
+`async function f() {
+   var result = await fetch('http://yahoo.com);

```

Comment:
I think we should default to `const` over `var`. But a `const` can't be reassigned, so you'd have to not use the name `result` twice if there is more than one awaited result.

Actually, the variable is unnecessary here since `return res(await fetch())` is closer to what the original code did.